content/blog/one-year-of-excalidraw/index.md

```diff
@@ -0,0 +1,178 @@
+
+### 💾 File system integration and file handling
+
+In Excalidraw, we use the [browser-nativefs](https://github.com/GoogleChromeLabs/browser-nativefs) library to integrate with the file system of the operating system. This allows us to support a true open→edit→save workflow with proper over-saving and save-as on supported browsers, with a fallback to file uploads and downloads on other browsers. Read more about this feature in [tomayac](https://github.com/tomayac)'s [earlier article](/browser-nativefs/) on this blog. We also have [experimental support](https://web.dev/file-handling/#demo) for file type association, so that when you double-click an `.excalidraw` file in your file explorer, the Excalidraw PWA opens.
+
+### ⚙️ Gatsby plugin
+
+We also have a plugin for Gatsby that automatically converts links to saved Excalidraw drawings to inline SVG at build time. We are actually using it in this post for our charts. Implemented by [trevorblades](https://github.com/trevorblades) & [j-f1](https://github.com/j-f1), and you can find it under [@excalidraw/gatsby-embedder-excalidraw](https://github.com/excalidraw/gatsby-embedder-excalidraw).
+
+https://twitter.com/Vjeux/status/1257906664239333376
+
+### 🌒 Dark Mode
+
+We also launched virtual blackboard. Thanks to [@xixixao](https://twitter.com/xixixao) for building this. https://twitter.com/Msieur_Jo/status/1245288337897914373
+
+### 🏹 Arrow Binding
+
+This was also one of the most awaited features. Thanks to [@xixixao](https://twitter.com/xixixao) for building this. https://twitter.com/excalidraw/status/1292403762427039744
+
+### 📦 npm package
+
+One of the last things we've introduced this year was a completely new npm package, available at [`@excalidraw/excalidraw`](https://www.npmjs.com/package/@excalidraw/excalidraw). A long time in the making (thanks to [@aakansha1216](https://twitter.com/aakansha1216) for most of the work), this package allows you to easily embed Excalidraw as a React component into your apps.
+
+## Excalidraw in the news
+
+- Featured twice on Hacker News:
+  - https://news.ycombinator.com/item?id=23525648
+  - https://news.ycombinator.com/item?id=22663435
+- As an example on **web.dev** article: https://web.dev/browser-nativefs/
+- A few blog posts:
+  - https://pakstech.com/blog/draw-diagrams/
+  - https://dev.to/ndsn/why-excalidraw-is-mightier-than-the-pen-and-the-sword-329f
+- On Product Hunt: https://www.producthunt.com/posts/excalidraw
+- React Europe talk: https://www.youtube.com/watch?v=fix2-SynPGE
+- Used to illustrate an O'Reilly Book: https://www.amazon.com/dp/1492057096
+- Another book: https://twitter.com/dchest/status/1264237749642637312
+- Few companies also integrated Excalidraw in the product
+  - [HackerRank](https://blog.hackerrank.com/virtual-whiteboarding-for-system-design-interviews/)
+  - [Lobelia Earth](https://twitter.com/lobeliaearth/status/1275073557484244992)
+
+https://twitter.com/wietsevenema/status/1253752608671621124
```

Comment:
shouldn't this be part of drawings ?

Reply:
it should be under the O'Reilly Book. I'll put it there.